ios/RNCallKeep/RNCallKeep.m

```diff
@@ -133,8 +136,22 @@ + (void)initCallKitProvider {
     }
 }
 
++ (void)setup:(NSDictionary *)options {
+    RNCallKeep *callKeep = [RNCallKeep allocWithZone: nil];
+    [callKeep setup:options];
+    isSetupNatively = YES;
+}
```

Comment:
Simply allocate the singleton and call setup in there, setting isSetupNatively makes the library ignore further calls to setup from react native.